source/operations/network-encryption.rst

```diff
@@ -69,16 +72,29 @@ Enabling TLS
         private.key
         public.crt
 
-   You can use the MinIO :minio-git:`certgen <certgen>` to mint self-signed certificates for enabling TLS for evaluating MinIO with TLS enabled.
-   For example, the following command generates a self-signed certificate with a set of IP and DNS SANs associated to the MinIO Server hosts:
+   You can use the MinIO :minio-git:`certgen <certgen>` to mint self-signed certificates for evaluating MinIO with TLS enabled.
+   For example, the following command generates a self-signed certificate with a set of IP and DNS Subject Alternate Names (SANs) associated to the MinIO Server hosts:
 
    .. code-block:: shell
 
       certgen -host "localhost,minio-*.example.net"
 
-   You can place the generated ``public.crt`` and ``private.key`` into the ``/.minio/certs`` directory to enable TLS for the MinIO deployment.
+   Place the generated ``public.crt`` and ``private.key`` into the ``/.minio/certs`` directory to enable TLS for the MinIO deployment.
    Applications can use the ``public.crt`` as a trusted Certificate Authority to allow connections to the MinIO deployment without disabling certificate validation.
 
+   If you are reconfiguring an existing deployment that did not previously have TLS enabled, update ``MINIO_VOLUMES`` to specify ``https`` instead of ``http``.
+   You may also need to update URLs used by applications or clients.
+
+   Self-signed Certificates
+   ~~~~~~~~~~~~~~~~~~~~~~~~
+
+   For self-signed and internally signed certs, place the appropriate CA certificate in a ``{HOME}/.minio/CAs`` directory:
+
+   .. code-block:: shell
+
+      ${HOME}/.minio/CAs
+        myCA.pem
```

Comment:
What should this filename be? I'm not certain it's actually a pem.

Reply:
Also depends - if it's self-signed, it's going to be a `.crt`.

We might need to confirm w/ engineering what we expect the keys to be. `.pem` is a valid wrapper, but I'm not sure if we require `.crt`

Reply:
Poking at the code, I _think_ we expect `.crt`. Will confirm.